pool/position.gno

```diff
@@ -26,11 +44,11 @@ func positionGetKey(
 // positionUpdateWithKey updates a position in the pool and returns the updated position.
 func (pool *Pool) positionUpdateWithKey(
```

Comment:
didn't we agree on changing this method to something else? I don't quite remember but I think we did talk about this

Reply:
we discussed about making function as method, and this function is already method.